data/templates/firewall/upnpd.conf.j2

```diff
@@ -117,9 +117,7 @@ clean_ruleset_threshold=10
 clean_ruleset_interval=600
 
 # Anchor name in pf (default is miniupnpd)
-# Something wrong with this option "anchor", comment it out
-#   vyos@r14# miniupnpd -vv -f /run/upnp/miniupnp.conf
-#   invalid option in file /run/upnp/miniupnp.conf line 74 : anchor=VyOS
+# pf(4) does not apply to Linux, hence must be commented out here.
 #anchor=VyOS
```

Comment:
Should we delete it ?

Reply:
I contemplated just deleting it, but figured all those commented out ones are present here to be as close to the stock example miniupnpd.conf as possible (easier to tell when new options are added or something changes which may require attention). 

IMO it's fine like this, and consistent with the remainder of the file.

Reply:
I went through and synced up `upnpd.conf.j2` with miniupnp's current `miniupnpd.conf`. 

I added a header and footer around the `anchor=` and 4 other pf-specific options which aren't usable on VyOS. That work for you @dmbaturin ?